src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -19,12 +20,21 @@
         :container-style style/empty-container-style}]
       (into [rn/view {:style style/my-accounts-container}]
             (map (fn [{:keys [color address] :as account}]
-                   [quo/account-item
-                    {:account-props (assoc account :customization-color color)
-                     :on-press      #(rf/dispatch [:wallet/select-send-address
-                                                   {:address   address
-                                                    :recipient account
-                                                    :stack-id  :screen/wallet.select-address}])}]))
+                   (let [network-short-names (map network-utils/network->short-name
```

Comment:
Isn't the computation of `address` identical to the other file in `from/view.cljs`? We should probably avoid this duplication if it's a true one.

Reply:
Done

Reply:
Awesome, thank you applying this change.